pyrokinetics/pyro.py

```diff
@@ -1338,12 +1322,20 @@ def load_local_species(self, psi_n: float, a_minor: Optional[float] = None) -> N
                     "load_global_equilibrium."
                 )
 
+        self.local_norm = Normalisation.from_kinetics(
+            self.kinetics,
+            psi_n=psi_n,
+            convention=self.local_norm.nocos.name,
+            lref=a_minor,
+            bref=self.local_norm.bref,
```

Comment:
Here we need to reuse `bref` from that calculated during `load_geometry` above. Looking at this with fresh eyes, this does put a dependency on the order in which `load_geometry` and `load_local_species` should be called.

Reply:
Yes this is true. We should add a check to make sure `local_geometry` is in before `local_species` is loaded